src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -864,6 +885,25 @@ def pin_subvolume_group(self, **kwargs):
             ret = self.volume_exception_to_retval(ve)
         return ret
 
+    def subvolume_group_exists(self, **kwargs):
+        volname = kwargs['vol_name']
+        ret = 0, "", ""
+        volume_exists = False
+
+        try:
+            with open_volume(self, volname) as fs_handle:
+                volume_exists = True
+                res = has_subdir(fs_handle, self.volspec.base_dir, filter_entries=[
+                                 dir.encode('utf-8') for dir in self.volspec.INTERNAL_DIRS])
+                if res:
+                    ret = 0, "Subvolumegroup(s) exist", ""
+                else:
+                    ret = 1, "No subvolumegroup exist", ""
```

Comment:
I think the caller is treating any non-negative return value as success and converting it to 0. So it doesn't make sense to return 1 here. We should return 0 and rely on string msg ?